spec/javascripts/accordion-with-descriptions-spec.js

```diff
@@ -0,0 +1,181 @@
+describe('An accordion with descriptions module', function () {
+  "use strict";
+
+  var $element;
+
+  beforeEach(function() {
+    $element = $('<div class="subsections js-hidden" data-module="accordion-with-descriptions">\
+        <div class="subsection-wrapper">\
+          <div class="subsection">\
+            <div class="subsection__header">\
+              <h2 class="subsection__title">Subsection title in here</h2>\
+              <p class="subsection__description">Subsection description in here</p>\
+            </div>\
+            <div class="subsection__content" id="subsection_content_0">\
+              <ul class="subsection__list">\
+                <li class="subsection__list-item">\
+                  <a href="">Subsection list item in here</a>\
+                </li>\
+              </ul>\
+            </div>\
+          </div>\
+          <div class="subsection">\
+            <div class="subsection__header">\
+              <h2 class="subsection__title">Subsection title in here</h2>\
+              <p class="subsection__description">Subsection description in here</p>\
+            </div>\
+            <div class="subsection__content" id="subsection_content_1">\
+              <ul class="subsection__list">\
+                <li class="subsection__list-item">\
+                  <a href="">Subsection list item in here</a>\
+                </li>\
+              </ul>\
+            </div>\
+          </div>\
+        </div>\
+      </div>');
```

Comment:
Only a minor thing.. Do we need all the backslashes? It makes it just a little tiny bit harder to read and wouldn't having the markup over multiple lines to closer to real life anyway?